hyperdome_server/web/share_mode.py

```diff
@@ -65,123 +66,114 @@ def __init__(self, common, web):
         self.web = web
 
         web.app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-        web.app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///./therapists.db'
+        web.app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///./counselors.db'
         self.db = SQLAlchemy(web.app)
         self.define_routes()
 
         self.bcrypt = Bcrypt(web.app)
         login_manager.init_app(web.app)
-        login_manager.login_view = "therapist_signin"
+        login_manager.login_view = "counselor_signin"
         login_manager.session_protection = None
-        self.therapists_available = []
-        self.connected_therapist = dict()
-        self.connected_guest = dict()
+        self.counselors_available = dict()
+        self.active_chat_user_map = dict()
         self.pending_messages = dict()
         self.user_class = get_user_class_from_db_and_bcrypt(self.db,
                                                             self.bcrypt)
 
-    def define_routes(self):
-
-        @self.web.app.before_request
-        def before_request():
-            user = load_user(request.headers.get("username", ""))
-            if user and user.is_correct_password(
-                    request.headers.get("password", "")):
-                login_user(user)
+        self.info = {'name': 'hyperdome',
+                     'version': self.common.version,
+                     'online': str(len(self.counselors_available))}
```

Comment:
Might want "online" to be a property instead of part of self.info and having to re-register the number of online users when you think you might need to use the variable

Reply:
"property" as in `@property`

Reply:
Online isn't currently filled or used at all, but I like this suggestion.

Leaving it for now.